### ISSUE TITLE
Accessibility - "Why Develop a Social Compliance System" Screen - Incorrect Search Button Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#28]
The " Why Develop a Social Compliance System" screen has incorrect Search button text. The text currently is "greater than Why Develop a Social Compliance System" and should be "search".

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
Thanks, for making the search icon say it's a search button.
